src/dotnet/Users.Service.Migration/Migrations/20220413141435_AddUserProfile.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace ActualChat.Users.Migrations.Migrations
+{
+    public partial class AddUserProfiles : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "UserProfiles",
+                columns: table => new
+                {
+                    Id = table.Column<string>(type: "text", nullable: false),
+                    Version = table.Column<long>(type: "bigint", nullable: false),
+                    Status = table.Column<short>(type: "smallint", nullable: false)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_UserProfiles", x => x.Id);
+                });
+
+            migrationBuilder.Sql(@"
```

Comment:
I really don't know how to implement compatible migration for Pgsql, MySQl  - could you please check this (look for the solution)

Reply:
At least we need to know how to deal with that

Reply:
it was decided to skip for now